protected/migrations/m170823_064156_cera_ethnic_groups.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class m170823_064156_cera_ethnic_groups extends OEMigration
+{
+	public function safeUp()
+	{
+        /**
+         * @var $ethnicity EthnicGroup
+         * @var $patient Patient
+         */
+	    // Save all ethnic groups so the original values can be versioned.
+        $ethnicGroups = EthnicGroup::model()->findAll();
+        foreach ($ethnicGroups as $ethnicity) {
+            $ethnicity->save();
```

Comment:
This is just saving the records to add versioning. Probably doesn't matter too much if it succeeds or fails.